src/config/defaults.ts

```diff
@@ -19,6 +19,14 @@ export interface FileOptions {
   readonly manifestFileName?: string;
 }
 
+export interface ShortcutOptions {
+  readonly name: string;
+  readonly short_name?: string;
+  readonly description?: string;
+  readonly url: string;
+  readonly icon?: string | Array<string>;
```

Comment:
There is no style guide in this package. But all arrays are written as `type[]`.
It there a good reason to prefer one syntax over another?